package.json

```diff
@@ -37,6 +37,7 @@
     "node-slack": "0.0.7",
     "node-uuid": "1.4.7",
     "nodemailer": "1.11.0",
+    "opencollective-jobs": "github:opencollective/opencollective-jobs#be6d0a2014e36cdd073a5dda2b1f81d71ce07a7d",
```

Comment:
that references a particular commit.  if we remove it, we'll get whatever the `master` branch happens to be when `npm install` is run.  so, no, we  want to keep it until we start doing releases for that project